### ISSUE TITLE
mvn problem

### ISSUE THREAD

**Issue by LeiRui** [thulab/iotdb-benchmark#245]
With the master branch, `mvn clean package -Dmaven.test.skip=true` fails:

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for iotdb-benchmark 0.0.1:
[INFO]
[INFO] iotdb-benchmark .................................... SUCCESS [  0.142 s]
[INFO] Benchmark Core ..................................... FAILURE [ 37.602 s]
[INFO] Benchmark IoTDB 0.12 ............................... SKIPPED
[INFO] Benchmark IoTDB 0.11 ............................... SKIPPED
[INFO] Benchmark IoTDB 0.10 ............................... SKIPPED
[INFO] Benchmark IoTDB 0.09 ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  37.859 s
[INFO] Finished at: 2022-01-24T15:26:29+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project core: Could not resolve dependencies for project cn.edu.tsinghua:core:jar:0.0.1: Failed to collect dependencies at org.glassfish.jaxb:jaxb-runtime:jar:2.3.0-b170127.1453 -> org.glassfish.jaxb:jaxb-core:jar:2.3.0-b170127.1453 -> javax.xml.bind:jaxb-api:jar:2.3.0-b161121.1438: Failed to read artifact descriptor for javax.xml.bind:jaxb-api:jar:2.3.0-b161121.1438: Could not transfer artifact javax.xml.bind:jaxb-api:pom:2.3.0-b161121.1438 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [releases.java.net (http://maven.java.net/content/repositories/releases/, default, releases+snapshots), shapshots.java.net (http://maven.java.net/content/repositories/snapshots/, default, releases+snapshots), jvnet-nexus-staging (http://maven.java.net/content/repositories/staging/, default, releases+snapshots), netbeans (http://bits.netbeans.org/nexus/content/groups/netbeans, default, releases)] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :core
```

**Comment by stormbroken** [comment]
Oh, From my perspective, I think it may because the problem of your maven repo. I suggest you can package without using mirror.

**Comment by cornmonster** [comment]
This is caused by a feature introduced in maven version 3.8.1, see this post for more detailed information: https://stackoverflow.com/a/68394404/7098025

**Comment by HTHou** [comment]
Fixed by this PR? @stormbroken 

https://github.com/thulab/iotdb-benchmark/pull/255

**Comment by stormbroken** [comment]
Yes, this problem has been fixed by #255
